### ISSUE TITLE
oadp backup wrong ns

### ISSUE THREAD

**Issue by lance5890** [openshift/openshift-docs#56873]
### Which section(s) is the issue in?
when create oadp backup , its ns should be consistent with backupStorageLocations, as the code show:
https://github.com/openshift/velero/blob/6f1b238003df467715f8f16e7a540c1a9f0cebd7/pkg/controller/backup_sync_controller.go#L134-L196

the clusterBackups ns(L196) should be the same with ListBackupStorageLocations(L134)

### What needs fixing?
we should use the backupStorageLocations in the same namespace openshift-adp

**Comment by anarnold97** [comment]
@kaovilai Please can you tell us if this will require a doc ticket?
Thanks

**Comment by kaovilai** [comment]
I can't see any issue with the code. If this bug is filed as part of because of [diff](https://github.com/openshift/openshift-docs/pull/56874/files#diff-6a3b74be2b9787c27edb6f7b56cd2de83a5a32f5c2d7af0229d9841f9ede3b4cL37-R38) at #56874 below
```diff
-NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
-velero-sample-1   Available   11s              31m
+NAMESPACE       NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
+openshift-adp   velero-sample-1   Available   11s              31m
```
Please see that the Namespace was not mentioned in the first output. Namespace was never "inconsistent"/

The name of BackupStorageLocation is still velero-sample-1, just shifted to the right by one column in your diff.

**Comment by kaovilai** [comment]
No doc ticket is required here.

**Comment by kaovilai** [comment]
L196 and L134 use the same string from the same variable `c.namespace`.
There is nothing that modifies this variable between the said lines

**Comment by kaovilai** [comment]
The linked doc PR is valid. Maybe this issue title should be renamed to
```diff
-oadp backup wrong ns
+oadp: `oc get backupstoragelocations` should include `-n openshift-adp`
```

It is an enhancement over prior doc but not a bug/error imo.

**Comment by lance5890** [comment]
> I can't see any issue with the code. If this bug is filed as part of because of [diff](https://github.com/openshift/openshift-docs/pull/56874/files#diff-6a3b74be2b9787c27edb6f7b56cd2de83a5a32f5c2d7af0229d9841f9ede3b4cL37-R38) at #56874 below
> 
> ```diff
> -NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
> -velero-sample-1   Available   11s              31m
> +NAMESPACE       NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
> +openshift-adp   velero-sample-1   Available   11s              31m
> ```
> 
> Please see that the Namespace was not mentioned in the first output. Namespace was never "inconsistent"/
> 
> The name of BackupStorageLocation is still velero-sample-1, just shifted to the right by one column in your diff.

yeah , what i mean is the ns of  backup should be the same with the ns of  backupstoragelocation

**Comment by lance5890** [comment]
> The linked doc PR is valid. Maybe this issue title should be renamed to
> 
> ```diff
> -oadp backup wrong ns
> +oadp: `oc get backupstoragelocations` should include `-n openshift-adp`
> ```
> 
> It is an enhancement over prior doc but not a bug/error imo.

done

**Comment by anarnold97** [comment]
Moved to [OADP-1403](https://issues.redhat.com/browse/OADP-1403)

**Comment by kalexand-rh** [comment]
Since the work to address this issue will be completed on the Jira that @anarnold97 linked, I'm closing this issue.

**Comment by lance5890** [comment]
> Since the work to address this issue will be completed on the Jira that @anarnold97 linked, I'm closing this issue.

but the linked PR is still open ,maybe u should merge this pr

**Comment by kaovilai** [comment]
Sounds like it will be tracked in https://issues.redhat.com/browse/OADP-1403 so guess not yet.

**Comment by lance5890** [comment]
> @kaovilai @lance5890 Is there any documentation work to do for this?

this has already been fixed
